content/posts/2023-05-25-rse-survey-2023-results.md

```diff
@@ -5,7 +5,7 @@ image: '/img/survey2023/thumbnail.jpg'
 date: 2023-05-25
 ---
 
-By Lieke de Boer and Thomas Pronk, NL-RSE
```

Comment:
@liekelotte -- very nice of you to include me as an author, but I think you've been pushing this blog post more than I have and deserve the recognition for it!